.github/workflows/databases-nightly.yml

```diff
@@ -0,0 +1,53 @@
+name: databases-nightly
+
+on:
+  pull_request: # TODO: Remove before merge
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+
+ mariadb:
+    runs-on: ubuntu-22.04
```

Comment:
indentation seem's wrong here? 2 space for each indent, right?

Reply:
Thanks. I fixed it. (Github Actions has still run the job interestingly)